tests/integration/test_cubepart.py

```diff
@@ -861,7 +861,7 @@ def it_provides_values_for_univariate_cat(self):
         assert strand.table_proportions == pytest.approx([0.6666667, 0.3333333])
         assert strand.title == "Registered Voters"
         assert strand.unweighted_bases == (15, 15)
-        assert strand.unweighted_counts == (10, 5)
+        assert strand.unweighted_counts.tolist() == [10, 5]
```

Comment:
`strand.unweighted_counts` is now type `ndarray` which doesn't directly allow comparison. When you say `assert array_x == array_y`, the `array == something` expression forms another array of boolean, and that has no truth value (not == True and also not == False). So the `.tolist()` is pretty good because it verifies the type as `ndarray` (tuple or list have no `.tolist()` method) as well as allow for a convenient `list` comparand.